### PR TITLE
Add home page and bugs urls to meta data.

### PR DESCRIPTION
**Pull request by fabricematrat**

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/charmrepo/105/
Test PASSed.

File: csclient/params/params.go [hunk 1]
```diff
@@ -221,6 +221,20 @@ type PermRequest struct {
 	Write []string
 }
 
+// HomePage holds the request/response of an id/meta/home-page PUT or GET request.
+// See https://github.com/juju/charmstore/blob/v4/docs/API.md#put-idmeta
+// See https://github.com/juju/charmstore/blob/v4/docs/API.md#get-idmetahome-page
+type HomePage struct {
```
Comment (urosj) [inline csclient/params/params.go]: Shouldn't this be HomePageRequest / Response as the rest of the perms?

File: csclient/params/params.go [hunk 2]
```diff
@@ -221,6 +221,20 @@ type PermRequest struct {
...
+	HomePage string
+}
+
+// BugsURL holds the request/response of an id/meta/bugs-url PUT or GET request.
+// See https://github.com/juju/charmstore/blob/v4/docs/API.md#put-idmeta
+// See https://github.com/juju/charmstore/blob/v4/docs/API.md#get-idmetabugs-url
+type BugsURL struct {
```
Comment (urosj) [inline csclient/params/params.go]: Ditto.

File: csclient/params/params.go (hunk 1 above)
Comment (fabricematrat) [inline csclient/params/params.go]: Except this is the same struct today but I could use 2 differents structs, not really sure it adds something today.

Comment (mhilton) [inline csclient/params/params.go]: I'm with @urosj on this one. PermRequest & PermResponse above makes it very clear what it going on. If we ever want to change one of them we can in a backwards compatible way. On a slightly meta level is there a reason this information shouldn't go in extra-info removing the need for this discussion at all?

**Pull request closed without merging**